dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -299,3 +299,13 @@ krew.sigs:
 secrets-store-csi-driver.sigs:
 - type: CNAME
   value: kubernetes-sigs-secrets-store-csi-driver.netlify.app.
+# https://github.com/kubernetes-sigs/cli-experimental docs (@pwittrock @seans3 @eddiezane @soltysh)
+# kustomize and kubectl share the same docs site
+kustomize.sigs:
+  type: CNAME
```

Comment:
I think this is meant to have a - before it as it is a list of structures (like the entry above)

Reply:
Ah, I see your discussion on slack now and that other entries do not 🙈

Reply:
:)